shell/platform/tizen/channels/settings_channel_linux.cc

```diff
@@ -0,0 +1,40 @@
+// Copyright 2021 Samsung Electronics Co., Ltd. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "settings_channel.h"
+
+#include <chrono>
+#include <ctime>
+
+namespace flutter {
+
+void SettingsChannel::Init() {
+  std::locale::global(std::locale(""));
```

Comment:
What does this call do?

Reply:
This is the code that applies the system locale.